README.md

```diff
@@ -1,17 +1,17 @@
 # Stampie
 
-[![Build Status](https://secure.travis-ci.org/henrikbjorn/Stampie.png)](http://travis-ci.org/henrikbjorn/Stampie)
+[![Build Status](https://secure.travis-ci.org/henrikbjorn/Stampie.png)](https://travis-ci.org/Stampie/Stampie)
```

Comment:
I would switch to the SVG version of the badge (and stop using the old `secure` subdomain)